documentation/milestones/individual_milestones/get_opportunities.md

```diff
@@ -0,0 +1,196 @@
+- [ ] **[Database (DB) replica](https://github.com/HHS/grants-equity/issues/):** Ensures parity between the set of opportunities returned by the new GET Opportunities endpoint and the legacy system. It also allows users to access the endpoint when there are outages on the legacy system.
+- [ ] **[Feature flag framework](https://github.com/HHS/grants-equity/issues/):** Enables us to deploy new features or changes without immediately exposing them to the public. 
+- [ ] **[API documentation](https://github.com/HHS/grants-equity/issues/):** Establishes a location and strategy for publishing information about the GET Opportunities endpoint (and future API endpoints) that users can reference when learning how to interact with the API.
+- [ ] **[API Versioning](https://github.com/HHS/grants-equity/issues/68):** Establishes a protocol for publishing breaking and non-breaking changes to the API.
+- [ ] **[API Security Planning](https://github.com/HHS/grants-equity/issues/):** Sets up minimum security standards to protect the API endpoint, such as API keys, rate limits, and security incident response protocols.
+
+*Are there any notable capabilities / milestones do NOT we expect to be in place at the beginning of work on this milestone?*
+
+- [ ] **Performance Testing Framework:** While performance should be an important consideration during the development of this initial API endpoint, a fully implemented performance and load testing framework is scoped for a later milestone.
+- [ ] **AuthN/AuthZ:** While the implementation of rate limiting or other API security measures may require some basic authentication, the full AuthN/AuthZ framework will be developed in a later milestone.
+
+### Open questions
+<!-- Optional -->
+
+- [x] None
+
+### Not doing
+<!-- Optional -->
+
+The following work will *not* be completed as part of this milestone:
+
+1. **User Interface:** Because this milestone is focused on the API endpoint, it will not include delivering a user interface for non-technical users to access a list of opportunities. That work will be incorporated in the Search UI milestone instead.
+2. **Translating API Docs:** Translation of key documents will be covered in an upcoming milestone.
+
+## Integrations
+
+### Translations
+<!-- Required -->
+
+*Does this milestone involve delivering any content that needs translation?*
+
+Yes, portions of the API user guide and docs will need to be translated.
+
+*If so, when will English-language content be locked? Then when will translation be started and completed?*
+
+Timeline and strategy for translation is still TBD.
+
+### Services going into PROD for the first time
+<!-- Required -->
+
+*This can include services going into PROD behind a feature flag that is not turned on.*
+
+1. **API:** This milestone is the official release of the `beta.grants.gov/api`
+2. **Replica Database:** A replica of relevant tables from the legacy database, which also contains the new `beta.grants.gov` data model
```

Comment:
What does this mean? "which also contains the new `beta.grants.gov` data model"

Reply:
I might be the confused one here. My understanding of a replica DB is that it's identical to the source that it's replicating -- therefore, there would not be any new tables in the replica (such as tables for the beta.grants.gov data model) that aren't in the source, existing DB. I believe any new tables would need to be created in their own database. 

My knowledge might be out of date here though! As long as we're clear in what we're planning.

Reply:
Ahh gotcha! I'm not sure that has been decided just yet i.e. whether the updated data model will live in a separate database or within a new schema of the replica database. I'll update this language to allow for both possibilities.

Reply:
Made the change in [this commit](https://github.com/HHS/grants-equity/pull/249/commits/93bf2532781b0d4e4820f0d98906ba37713199bb)